draft-pauly-ohai-svcb-config.md

```diff
@@ -168,34 +180,42 @@ still need to ensure that they are not being targeted with unique
 key configurations that would reveal their identity. See {{security}} for
 more discussion.
 
-### Handling Oblivious DoH Configurations
+# Configuration Well-Known URI
+
+Once a client has discovered that a service is available as an oblivious
+target, it needs to know the key configuration before sending oblivious
+requests.
+
+This document defines a well-known URI {{!RFC8615}}, "oblivious-configs",
+that allows a target to host its configurations.
 
-Oblivious DoH was originally defined in
-{{?ODOH=I-D.draft-pauly-dprive-oblivious-doh}}. This version of
-Oblivious DoH uses a different key configuration format than
-generic Oblivious HTTP. SVCB records using the "dns" scheme
-can include one or more `ObliviousDoHConfig` structures 
-using the "odoh-configs" parameter.
+The URI is constructed using the TargetName in the SVCB record.
```

Comment:
Rather than use the TargetName directly, should clients chase TargetNames down to a terminal node in the SVCB chain? For example, if _dns.resolver.arpa targets doh.example, and doh.example targets jk-this-doh.example, presumably the target the client should use is jk-this-doh.example, right?

Reply:
Clarified that this should be off of the ServiceMode.